x-pack/plugins/security_solution/scripts/endpoint/agent_emulator/services/endpoint_response_actions.ts

```diff
@@ -319,11 +321,24 @@ const getOutputDataIfNeeded = (action: ActionDetails): ResponseOutput => {
       } as ResponseOutput<ResponseActionGetFileOutputContent>;
 
     case 'execute':
+      const executeOutput: Partial<ResponseActionExecuteOutputContent> = {
+        output_file_id: getFileDownloadId(action, action.agents[0]),
+      };
+
+      // Error?
+      if (commentUppercase.indexOf('EXECUTE:FAILURE') > -1) {
```

Comment:
the smallest of nits: `commentUppercase.includes('EXECUTE:FAILURE')` could be used